### DISCUSSION TITLE
WIS - Absprung ins WebGIS

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#114] [Q&A]
Hallo Jürgen!

Man kann über URL Parameter ja auch Werkzeuge aufrufen, z.B. webgis.tools.editing.edit. Ist es möglich beim Edit-Werkzeug direkt mitzugeben, ob ein Neues Objekt angelegt oder ein Bestehendes Objekt bearbeitet werden soll? Wäre es dann auch möglich die Kategorie mitzugeben (d.h. in unserem Fall ob es ein Punkt, Linie oder Fläche ist)?

Also, kann man über URL Parameter direkt z.B. zu diesem Schritt springen:

![wis_edit](https://github.com/gis-eni/webgis-forum/assets/122986367/101a275f-4799-4d78-b361-202e4d587620)

Und wäre es dann möglich über URL Parameter dann auch ein Attribut mitzugeben und zu verspeichern (z.B. WIS-ID)?

Das führende System für die Erstellung neuer WIS-Objekte ist die WIS-DB. Dort wird die ID angelegt. Aus dem WIS soll dann mittels URL-Aufruf die WIS-ID mitgegeben werden und im WebGIS das Editierwerkzeug mit "Neues Objekt anlegen" aufgerufen werden sowie die korrekte Kategorie (Punkt, Linie, Fläche). Der User soll dann nur das Objekt in der Karte setzen und auf Speichern klicken. Damit soll dann die Geometrie und die WIS-ID in der SDE gespeichert werden.

Lg, Andi

**Comment by Andreas-Lampert** [comment]
Vielen Dank Stephanie @fanie-mak für die Unterstützung! 
Beispiel-Link --> &tool=webgis.tools.editing.edit&editthemeid=Interne ThemeId&ed_WIS_ID=@SUB(WIS_ID)

**Comment by gis-eni** [comment]
Die Felder kannst du mit ``...&ed_[feldname]`` übergeben.

Das mit gleich auf ``Neues Objekt erstellen`` springen kann, ist momentan noch nicht möglich. Kommt vielleicht mir der neuen Edit Umgebung

**Comment by gis-eni** [comment]
Hilfe erweitern
